src/pages/sa-price-groups/forms/PriceGroupsForm.js

```diff
@@ -0,0 +1,161 @@
+import { Grid } from '@mui/material'
+import { useContext, useEffect } from 'react'
+import * as yup from 'yup'
+import FormShell from 'src/components/Shared/FormShell'
+import toast from 'react-hot-toast'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useInvalidate } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { useForm } from 'src/hooks/form'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { ControlContext } from 'src/providers/ControlContext'
+import { SaleRepository } from 'src/repositories/SaleRepository'
+import CustomNumberField from 'src/components/Inputs/CustomNumberField'
+
+export default function PriceGroupsForm({ labels, maxAccess, recordId }) {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const invalidate = useInvalidate({
+    endpointId: SaleRepository.PriceGroups.qry
+  })
+
+  const { formik } = useForm({
+    initialValues: {
+      recordId: null,
+      reference: '',
+      name: '',
+      minUnitPrice: '',
+      maxUnitPrice: ''
+    },
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
```

Comment:
check the validation 
![Screenshot 2024-07-31 110431](https://github.com/user-attachments/assets/84f7f03c-f573-4844-b9e2-9b76e2b37e2f)

Reply:
done